gui/src/redux/slices/sessionSlice.ts

```diff
@@ -638,6 +638,7 @@ export const sessionSlice = createSlice({
       const modes = action.payload.isJetBrains
```

Comment:
there's a bug here, where "edit" is supposed to be **unavailable** for JB products. It results in Edit mode not being cycle-able on VS Code. The condition should be inverted. But I'll fix it upstream

Reply:
see https://github.com/continuedev/continue/pull/4987

Reply:
fix merged upstream